[not4land] PR dedicated for NVIDIA cuDNN benchmarking of resnet18

dzdang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #78155

Build instructions

1)
```
git clone https://github.com/pytorch/pytorch
```

2)
```
ghstack checkout <THIS_PR_URL>
```

3)
Make sure the libraries for cudnn are in LD_LIBRARY_PATH, then compile with
```
USE_EXPERIMENTAL_CUDNN_V8_API=ON MAX_JOBS=8 python setup.py develop
```

4)
Run resnet18 benchmark code from pytorch root directory
```
python test.py
```

dzdang committed:
PR dedicated for NVIDIA cuDNN benchmarking

Build instructions

1)
```
git clone https://github.com/pytorch/pytorch
```

2)
```
ghstack checkout <THIS_PR_URL>
```

3)
Make sure the libraries for cudnn are in LD_LIBRARY_PATH, then compile with
```
USE_EXPERIMENTAL_CUDNN_V8_API=ON MAX_JOBS=8 python setup.py develop
```

4)
Run resnet18 benchmark code from pytorch root directory
```
python test.py
```

[ghstack-poisoned]

jerryzh168 commented:
is this not for land? please add a tag [not4land] if that is the case

nickiverson commented:
It  seems that LD_LIBRARY_PATH is not necessary. I used the following command to build:
 CUDNN_LIB_DIR=<PATH_TO_CUDNN_DIR> CUDNN_INCLUDE_DIR=<PATH_TO_CUDNN_INCLUDE_DIR> USE_EXPERIMENTAL_CUDNN_V8_API=ON MAX_JOBS=48 python setup.py develop

dzdang committed:
Update on "[not4land] PR dedicated for NVIDIA cuDNN benchmarking of resnet18"


Build instructions

1)
```
git clone https://github.com/pytorch/pytorch
```

2)
```
ghstack checkout <THIS_PR_URL>
```

3)
Make sure the libraries for cudnn are in LD_LIBRARY_PATH, then compile with
```
USE_EXPERIMENTAL_CUDNN_V8_API=ON MAX_JOBS=8 python setup.py develop
```

4)
Run resnet18 benchmark code from pytorch root directory
```
python test.py
```

[ghstack-poisoned]
